config/initializers/summary_document_generator.rb

```diff
@@ -0,0 +1,5 @@
+SummaryDocumentGenerator.host = if Rails.env.development? || Rails.env.test?
```

Comment:
This isn't just the host. Perhaps we should rename it `uri`? Also, the check for dev or test environments is overkill in my opinion. We can just reduce it to `fetch` with the default you supplied across all environments. It will be apparent enough in production-like environments if it's misconfigured.